frame/staking/src/pallet/mod.rs

```diff
@@ -439,6 +476,61 @@ pub mod pallet {
 		ValueQuery,
 	>;
 
+	/// The nominator count each `ExposurePage` is capped at.
+	///
+	/// This cannot be greater than `T::MaxNominatorRewardedPerValidator`.
+	#[pallet::storage]
+	pub type ExposurePageSize<T> = StorageValue<_, u32, OptionQuery>;
```

Comment:
Further down the road, since we want to have the pallet be entirely bounded, we probably need to use these as `BoundedVec<_>` bounds. 

Shouldn't we therefore make them just be `Get<_>` types from now?